packages/grid/_modules_/grid/hooks/utils/useGridScrollbarSizeDetector.tsx

```diff
@@ -22,30 +23,33 @@ export function getScrollbarSize(doc: Document, element: HTMLElement): number {
   return scrollbarSize;
 }
 
-export function useGridScrollbarSizeDetector(apiRef: GridApiRef, scrollBarSizeProp?: number) {
+export function useGridScrollbarSizeDetector(
+  apiRef: GridApiRef,
+  { scrollbarSize }: Pick<GridComponentProps, 'scrollbarSize'>,
+) {
   const logger = useLogger('useGridScrollbarSizeDetector');
   const [detectedScrollBarSize, setDetectedScrollBarSize] = React.useState<number>(0);
   const [, setGridState] = useGridState(apiRef);
   const hasColumns = useGridSelector(apiRef, allGridColumnsSelector).length > 0;
 
   const detectScrollbarSize = React.useCallback(() => {
-    let scrollbarSize = 0;
+    let sbSize = 0;
```

Comment:
Does shortcut help? I would personally vote against:

```suggestion
    let measuredScrollbarSize = 0;
```